src/main/java/com/avioconsulting/mule/logger/internal/config/CustomLoggerConfiguration.java

```diff
@@ -345,5 +380,17 @@ public void initialise() throws InitialisationException {
       throw new InitialisationException(createStaticMessage(
           "Encryption Algorithm must be provided if encryption password is being supplied"), this);
     }
+    flowLogConfigs.forEach(flowLogConfig -> {
+      if (flowLogConfig.getMessageExpressionText() == null
+          && flowLogConfig.getAttributesExpressionText() == null) {
+        try {
+          throw new InitialisationException(createStaticMessage(
+              "One of attributesExpressionText or messageExpressionText must be defined in flow-logs-config"),
```

Comment:
I think "One of ..." would mean like those two properties are mutually exclusive, which is not the case. We can change it to "'attributesExpressionText' and 'messageExpressionText' cannot be empty, at least one must be specified."

Reply:
Updated Error message